src/sentry/features/__init__.py

```diff
@@ -50,17 +50,21 @@
 default_manager.add('organizations:create')
 
 # Organization scoped features
+default_manager.add('organizations:advanced_search', OrganizationFeature)  # NOQA
```

Comment:
I think we generally add a default in https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L790 as well